File: java_out/HelloWorld.java
```java
public class HelloWorld {
    public static void main(String[] args) {
        System.out.println("Hello, World from VB!");
        System.out.println("Hello, My name is Yogeshwar!");
        System.out.println("Hello, I am done successfully!");
        System.out.println("Hello, I am done successfully!");
        System.out.println("Hello, I am done successfully!");
        System.out.println("Hello, World from VB!");
        System.out.println("Hello, My name is Yogeshwar!");
        System.out.println("Hello, I am done successfully!");
        System.out.println("Hello, I am done successfully!");
        System.out.println("Hello, I am done successfully!");
        System.out.println("Hello, I am done successfully!");
        System.out.println("Hello, World from VB!");
        System.out.println("Hello, My name is Yogeshwar!");
        System.out.println("Hello, I am done successfully!");
        System.out.println("Hello, I am done successfully!");
        System.out.println("Hello, I am done successfully!");

        // Call calculator
        Calculator.runCalculator();
    }
}

class Calculator {
    public static void runCalculator() {
        double a, b;
        String op;

        System.out.println("Enter first number:");
        a = Double.parseDouble(System.console().readLine());

        System.out.println("Enter second number:");
        b = Double.parseDouble(System.console().readLine());

        System.out.println("Enter operator (+, -, *, /):");
        op = System.console().readLine();

        if (op.equals("+")) {
            System.out.println("Result: " + (a + b));
        } else if (op.equals("-")) {
            System.out.println("Result: " + (a - b));
        } else if (op.equals("*")) {
            System.out.println("Result: " + (a * b));
        } else if (op.equals("/")) {
            System.out.println("Result: " + (a / b));
        } else {
            System.out.println("Invalid operator!");
        }
    }
}
```

File: vb_src/HelloWorld.vb
Module HelloWorld
    Sub Main()
        Console.WriteLine("Hello, World from VB!")
        Console.WriteLine("Hello, My name is Yogeshwar!")
        Console.WriteLine("Hello, I am done successfully!")
        Console.WriteLine("Hello, I am done successfully!")
        Console.WriteLine("Hello, I am done successfully!")
        Console.WriteLine("Hello, World from VB!")
        Console.WriteLine("Hello, My name is Yogeshwar!")
        Console.WriteLine("Hello, I am done successfully!")
        Console.WriteLine("Hello, I am done successfully!")
        Console.WriteLine("Hello, I am done successfully!")
        #this is chagnes
        
        hjgjhffkjghkljfgfhjf

        ' Call calculator
        Calculator.RunCalculator()
    End Sub
End Module

Module Calculator
    Public Sub RunCalculator()
        Dim a, b As Double
        Dim op As String

        

        Console.WriteLine("Enter first number:")
        a = Double.Parse(Console.ReadLine())

        Console.WriteLine("Enter second number:")
        b = Double.Parse(Console.ReadLine())

        Console.WriteLine("Enter operator (+, -, *, /):")
        op = Console.ReadLine()

        If op = "+" Then
            Console.WriteLine("Result: " & (a + b))
        ElseIf op = "-" Then
            Console.WriteLine("Result: " & (a - b))
        ElseIf op = "*" Then
            Console.WriteLine("Result: " & (a * b))
        ElseIf op = "/" Then
            Console.WriteLine("Result: " & (a / b))
        Else
            Console.WriteLine("Invalid operator!")
        End If
    End Sub
End Module
